ginivision/src/main/java/net/gini/android/vision/camera/CameraActivity.java

```diff
@@ -605,6 +606,11 @@ public void onDocumentAvailable(@NonNull Document document) {
         }
     }
 
+    @Override
+    public void onPaymentDataAvailable(@NonNull final PaymentData paymentData) {
+
```

Comment:
This method should be implemented, right?

Reply:
Yes, I will add to the documentation that this has to be overriden in a subclass to get the payment data with the Screen API.